py4j-java/src/main/java/py4j/ThreadUtil.java

```diff
@@ -0,0 +1,31 @@
+package py4j;
```

Comment:
I think you don't have to bother with creating a util here. Let's just fix them inplace at `GatewayServer`, `ClientServerConnection`, and let's name them.